apps/daengle/src/pages/login/index.tsx

```diff
@@ -1,9 +1,16 @@
 import styles from './index.module.scss';
 
-export default function Login(): JSX.Element {
+export default function Login() {
+  const REST_API_KEY = process.env.NEXT_PUBLIC_REST_API_KEY;
+  const REDIRECT_URI = process.env.NEXT_PUBLIC_REDIRECT_URI;
+  const kakaoAuthUrl = `https://kauth.kakao.com/oauth/authorize?response_type=code&client_id=${REST_API_KEY}&redirect_uri=${REDIRECT_URI}&response_type=code`;
+
```

Comment:
상수는 컴포넌트 코드 밖으로 빼 주세요!
```suggestion
const REST_API_KEY = process.env.NEXT_PUBLIC_REST_API_KEY;
const REDIRECT_URI = process.env.NEXT_PUBLIC_REDIRECT_URI;
const kakaoAuthUrl = `https://kauth.kakao.com/oauth/authorize?response_type=code&client_id=${REST_API_KEY}&redirect_uri=${REDIRECT_URI}&response_type=code`;

export default function Login() {}
```